src/dss/main.py

```diff
@@ -65,7 +65,15 @@ def initialize_command(kubeconfig: str) -> None:
     "--kubeconfig",
     help=f"Path to a Kubernetes config file. Defaults to the value of the KUBECONFIG environment variable, else to '{KUBECONFIG_DEFAULT}'.",  # noqa E501
 )
-def create_notebook_command(name: str, image: str, kubeconfig: str) -> None:
+@click.option("--no-gpu", is_flag=True, help="Create a notebook without GPU support.")
+@click.option(
+    "--gpu",
+    type=click.Choice(SUPPORTED_GPUS),
```

Comment:
Does this need to be case sensitive?
```suggestion
    type=click.Choice(SUPPORTED_GPUS, case_sensitive=False),
```